ui/app/controllers/jobs/job/index.js

```diff
@@ -31,6 +33,11 @@ export default class IndexController extends Controller.extend(
   sortDescending = false;
   activeTask = null;
 
+  /**
+   * @type {('current'|'historical')}
+   */
+  statusMode = 'current';
```

Comment:
`question`: Shouldn't this be a `tracked` property? I think despite being a `classic` controller `tracked` should be [backwards compatible](https://guides.emberjs.com/release/upgrading/current-edition/tracked-properties/#toc_backwards-compatibility). We do this in the `Tokens` controller I believe.

Reply:
`question`: Are you going to want to fetch data when this query parameter changes? If so, you'll need to update the associated `Route`?

Reply:
I can try to make this tracked, but we don't typically do this in the Nomad UI I've found so I've got some concerns with bringing glimmer concepts in.